packages/core/base/src/models/errors.ts

```diff
@@ -23,6 +23,7 @@ export const PaymentErrors = {
     PAYMENT_REJECTED_CARD_INCORRECT_NUMBER: "payments:payment-rejected.card-incorrect-number",
     TRANSACTION_ERROR_GENERIC: "payments:transaction-error.generic",
     CONTRACT_EXECUTION_REVERTED_GENERIC: "payments:contract-execution-reverted.generic",
+    MINTO_INVALID: "payments:mintTo.invalid",
```

Comment:
I would call this error `payments:user-wallet.invalid` since we are referring user-wallet in another errors up there